lib/Backends/OpenCL/CMakeLists.txt

```diff
@@ -1,6 +1,47 @@
+set(KERNELS_HDR ${GLOW_BINARY_DIR}/kernels_cl.inc)
```

Comment:
Minor request: The generated "*.inc" files are emitted into the root of the build directory. I wonder if it would be possible to put them e.g. under `<root of the build directory>/glow`, where we seem to write the rest of our auto-generated files like `AutoGenNodes.h`

Reply:
Yes, `${GLOW_BINARY_DIR}/glow/kernels_cl.inc ` sounds good! Check that you see other auto-generated files there.